media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Neither 1.1.15.1 nor 1.1.15.2 was not released (yet?).
So I'm not sure is it good to use it.
Anyway it is a last commit in repo after these versions.
Do you think `1.1.15.2_p20160530` is a good version string in this case?